examples/data-science/pandas/CDC_NDI_Mortality_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<p>Q: Shouldn't we leave the cell from above without the <code>loopback</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Theres no agreed upon way to do this for our notebooks but we have discussed adding a templating system that could autogenerate the prelude for our notebooks so we don't have to keep changing it. The integration tests find the prelude connection and replace it, so we could do the opposite during commit. For now though this is a development notebook used to get the example working.</p>